st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -98,13 +99,24 @@ def post(self, payload):
         }
 
         if action_alias_db.ack:
-            if 'format' in action_alias_db.ack:
+            try:
```

Comment:
Those two blocks would probably be a bit easier to read and follow if you refactored some of the functionality in a separate method so it would be less nesting.

In any case, not a blocker for me.